appveyor.yml

```diff
@@ -14,6 +14,7 @@ build_script:
   - Setup configure --user --ghc-option=-Werror --enable-tests
   - Setup build
   - Setup test unit-tests --show-details=streaming
+  # - Setup test package-tests --show-details=streaming --test-options=--skip-shared-library-tests
```

Comment:
I added it because I thought we could run the package tests soon.  The only failure is #3147.